cmd/lncli/walletrpc_active.go

```diff
@@ -142,24 +151,14 @@ func bumpFee(ctx *cli.Context) error {
 		return err
 	}
 
-	var confTarget, satPerByte uint32
```

Comment:
why remove this validation?

Reply:
The server already validates this, so this is just unnecessary maintenance. Change was suggested by @wpaulino and I liked it.